build/NuSpecs/ProjectReunionFrameworkPackage.nuspec

```diff
@@ -20,17 +20,17 @@
 
 <!--    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\sdk\Microsoft.ProjectReunion.winmd"/> -->
     <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\Microsoft.ProjectReunion.winmd"/>
-    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.FrameworkPackagePRI\Microsoft.ProjectReunion.pri"/>
+<!--    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.FrameworkPackagePRI\Microsoft.ProjectReunion.pri"/> -->
 
     <file target="lib\uap10.0" src="Intellisense\Microsoft.ProjectReunion.xml"/>
-    <file target="lib\uap10.0\Design" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion.Design\Microsoft.ProjectReunion.design.dll"/>
+    <file target="lib\uap10.0" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\Microsoft.ProjectReunion\Microsoft.ProjectReunion.dll"/>
 
 <!-- <file target="tools" src="$TOOLSDIR$\**\*.*"/> -->
 
 <!-- <file target="tools" src="VisualStudioToolsManifest.xml"/> -->
 
-    <file target="build" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftProjectReunionVersion.props"/>
-    <file target="buildTransitive" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftProjectReunionVersion.props"/>
+<!--     <file target="build" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftProjectReunionVersion.props"/> -->
+<!--     <file target="buildTransitive" src="$BUILDOUTPUT$\$BUILDFLAVOR$\$BUILDARCH$\FrameworkPackage\MicrosoftProjectReunionVersion.props"/> -->
     <file target="build\Common.targets" src="ProjectReunion-Nuget-Common.targets"/>
```

Comment:
We'll need this to get the end-to-end working. Will getting the integration through the nuget consumption path be a future PR?